config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -63,21 +63,29 @@ spec:
               admin_password_secret:
                 description: Secret where the admin password can be found
                 type: string
+                maxLength: 255
+                pattern: '^[a-zA-Z0-9][-a-zA-Z0-9]{0,253}[a-zA-Z0-9]$'
               postgres_configuration_secret:
                 description: Secret where the database configuration can be found
                 type: string
               old_postgres_configuration_secret:
                 description: Secret where the old database configuration can be found for data migration
                 type: string
+                maxLength: 255
+                pattern: '^[a-zA-Z0-9][-a-zA-Z0-9]{0,253}[a-zA-Z0-9]$'                
```

Comment:
@imedaouidene Could you please remove the trailing spaces here?

```
./config/crd/bases/awx.ansible.com_awxs.yaml
  Error: 75:71 [trailing-spaces] trailing spaces
 ```

That will make CI pass, then this will auto-merge.

Reply:
@rooftopcellist fixed in [c389fc6](https://github.com/ansible/awx-operator/pull/1549/commits/c389fc60cc5c00766608afdbe31862d37106f29b)